addon/model/model-range.ts

```diff
@@ -433,9 +461,57 @@ export default class ModelRange {
   }
 
   sameAs(other: ModelRange): boolean {
+    console.log('THISROOT', this.root);
+    console.log('OTHERROOT', other.root);
     return this.start.sameAs(other.start) && this.end.sameAs(other.end);
   }
 
+  compare(other: ModelRange): RangeComparison {
```

Comment:
range comparisons are tough, but I couldn't spot an oversight. well done :)

Reply:
Unfortunately I see now that the only usage is in its only test... :joy: 
I ended up not needing range comparisons at all

Reply:
the problem is that there are just so many cases that a caller will probably implement its own logic anyway based on the case it needs

Reply:
you either make assumptions about which cases are equivalent or you end up so many cases on the callsite that your logic is pretty much as complex as if you were to compare positions yourself